variables.tf

```diff
@@ -338,3 +338,28 @@ variable "capacity_rebalance" {
   type        = bool
   default     = null
 }
+
+# ASG instance refresh variables
+variable "asg_instance_refresh_strategy" {
+  description = "The strategy to use for instance refresh. The only allowed value is `Rolling`."
+  type        = string
+  default     = "Rolling"
+}
+
+variable "asg_instance_refresh_warmup" {
+  description = "The number of seconds until a newly launched instance is configured and ready to use."
+  type        = number
+  default     = 300
```

Comment:
it's better to use "null" in case of optional parameter so that terraform will not supply this value as default and use the provider's default